habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -1887,6 +1887,10 @@ class GymConfig(HabitatBaseConfig):
     action_keys: Optional[List[str]] = None
     achieved_goal_keys: List = field(default_factory=list)
     desired_goal_keys: List[str] = field(default_factory=list)
+    cull_visual_sensors: Optional[bool] = False
```

Comment:
How about `cull_unused_visual_sensors`

Reply:
How about a warning comment that explains the dependency issue. You can mention humanoid_detector_sensor and spot_head_stereo_depth_sensor.

Reply:
Also I would expect the comment for this flag to go above it, not below it.

Reply:
>Also I would expect the comment for this flag to go above it, not below it.

You right that in this file, only class docstrings are under the symbol, so I'll change accordingly.
However, comments above the attributes aren't recognized as docstrings.

![attribute_docstrings](https://github.com/user-attachments/assets/ceade5e9-b309-492d-9461-206574bebf16)